Content.Server/Spreader/SpreaderSystem.cs

```diff
@@ -87,18 +87,20 @@ private void OnGridUnpaused(EntityUid uid, SpreaderGridComponent component, ref
     private void OnGridInit(GridInitializeEvent ev)
     {
         var comp = EnsureComp<SpreaderGridComponent>(ev.EntityUid);
-
     }
 
     /// <inheritdoc/>
     public override void Update(float frameTime)
     {
         var curTime = _timing.CurTime;
 
-        // Check which grids are valid for spreading.
+        // Check which grids are valid for spreading
+        // Every map seems to have a single "SpreaderGrid" component
```

Comment:
```suggestion
        // Check which grids are valid for spreading
```

Don't need to muse in comments.